### DISCUSSION TITLE
FreeTube making request to Quad9 DNS

### DISCUSSION THREAD

**Discussion by johndoe432** [FreeTubeApp/FreeTube#2044] [Q&A]
When opening a video in FreeTube, I noticed that firstly it makes an almost instant request to a Google IP, but after that it starts connecting to 149.112.112.112, which is a Quad9 DNS IP.

Shouldn't FreeTube connect directly to YouTube IP while playing a video?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
FreeTube does not make requests to other domains then YouTube. I confirmed this with the use of Wireshark.

The Quad9 requests are likely related to your setup.

**Reply by aeropatron** [reply]
Not sure is related but,
while using Portmaster        https://github.com/Safing/portmaster   
Last days Portmaster was saying that FreeTube was trying to use something like secure? DNS, which could be good, but as Portmaster already does it I should disconnect that Freetube feature.

I ll reinstall it and add the message here.

**Comment by ghost** [comment]
Does freetube use secure dns of itself or it use system dns ? because if it use it's own dns i would like to ask an option to actually use DNS of the system.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
It just uses the system defined dns

**Reply by ghost** [reply]
> It just uses the system defined dns

Ok thanks (the title mislead me to think it's wasn't thanks for the answer) :+1:
